wp-content/themes/core/assets/css/src/theme/typography/headings.pcss

```diff
@@ -9,7 +9,7 @@
  * ------------------------------------------------------------------------- */
 
 .h1,
-.t-sink h1 {
+.t-sink > h1 {
```

Comment:
This is interesting, so sink now only applies these to immediates?

Reply:
Yes. it has to. Otherwise, the typographic sink styles would apply to an `<h2>` inside the the content, as well as the panel title. I don't know of a simpler way to say, apply to this heading, but not this other one inside a panel block.

Reply:
As a tangential win, it helps with one of our goals of preventing the "sink" typography usage getting "sprayed" across an entire layout/section/panel and then needing to apply resets/overrides to elements that shouldn't get the sink styles.

Reply:
Totally. I'm 95% on board. We'll have to see how it plays out though :)

Reply:
Agreed. A lot of these decisions are feeling a bit. Well....we'll give this a try. :grimacing: